TicTacToeApp/ComputerPlayer.swift

```diff
@@ -0,0 +1,25 @@
+
+import Foundation
+
+public class ComputerPlayer {
+
+  public init() {
```

Comment:
IMO, an empty `public init` that is required by the compiler might be one of the rare times a one-liner method is OK. As long as you are consistent, either way is fine with me. A quick look in a couple style-guides didn't show anything in either direction.

```swift
public init() {}
```